app.js

```diff
@@ -39,28 +43,48 @@ mongoose.connect(`mongodb://mongo_db:27017/${db.name}`,
     console.log(db)
     console.log('MongoDB Error:', err)
 })
+
+
 app.set('view engine', 'ejs')
 
 
 //Redis setup
 const redisClient = redis.createClient({host: 'redis'}) //Uses default PORT: 6379
 
+//DDoS prevention
+app.use((req, res, next) => {
+    if(tooBusy()) {
+        return res.sendStatus(503)
+    }
+    next()
+})
+
+//Bruteforce prevention
+const apiLimiter = rateLimit({
+    windowMs: 15 * 60 * 1000, //15 minutes
+    max: 100
+})
+
+app.use('/articles/', apiLimiter)
+
 //Pass Redis connection to middleware for easy access 
 app.use((req, res, next) => {
     req.redis = redisClient
     next()
 })
 
 //Express setup
-app.use(bodyParser.urlencoded({extended: true}))
+app.use(bodyParser.urlencoded({extended: true, limit: '1kb'}))
```

Comment:
mind extracting it to the top of the file as constant?

Reply:
Sure